src/assets/css/style.css

```diff
@@ -1,10 +1,140 @@
-body {
-  margin: 0;
-}
-
-h1 {
-  font-size: 30px;
-  font-family: -apple-system, blinkmacsystemfont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
-}
-
-/*# sourceMappingURL=style.css.map */
+body {
+  margin: 0;
+  background-color: #fafafa !important;
+}
+
+input {
+  border: 2px solid #f4f5f9;
+  outline: none;
+}
+
+h1 {
+  font-size: 30px;
+  font-family: -apple-system, blinkmacsystemfont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
```

Comment:
You repeated this font multiple times in your code. Consider applying it on the entire body or a better parent element instead of repeating yourself.